src/unit_tests/wazuh_modules/vulnerability_detector/test_wm_vuln_detector.c

```diff
@@ -224,6 +220,10 @@ static int build_test_cve_report(vu_report* report, int add_title, int add_condi
     report->ref_sources[1] = NULL;
     // Reported software
     os_strdup("libhogweed4", report->software);
+    if (1 == add_title){
```

Comment:
Not sure whether this is still necessary, as well as the `add_title`. I took a quick look at the code and the function `build_test_cve_report` is always called with the first parameter as `0`.

Reply:
You're right we don't need this anymore.